test/100-bridge-iptables.bats

```diff
@@ -666,3 +666,21 @@ EOF
     expected_rc=1 run_netavark --file ${TESTSDIR}/testfiles/ipv6-bridge.json setup $(get_container_netns_path)
     assert '{"error":"add ip addr to bridge: failed to add ipv6 address, is ipv6 enabled in the kernel?: Netlink error: Permission denied (os error 13)"}' "error message"
 }
+
+@test "$fw_driver - route metric from config" {
+    run_netavark --file ${TESTSDIR}/testfiles/metric.json setup $(get_container_netns_path)
+
+    run_in_container_netns ip -j route list match 0.0.0.0
+    default_route="$output"
+    assert_json "$default_route" '.[0].dst' == "default" "Default route was selected"
+    assert_json "$default_route" '.[0].metric' == "200" "Route metric set from config"
+}
+
+@test "$fw_drive - default route metric" {
+    run_netavark --file ${TESTSDIR}/testfiles/dualstack-bridge.json setup $(get_container_netns_path)
+
+    run_in_container_netns ip -j route list match 0.0.0.0
+    default_route="$output"
+    assert_json "$default_route" '.[0].dst' == "default" "Default route was selected"
+    assert_json "$default_route" '.[0].metric' == "100" "Default metric was chosen"
```

Comment:
Should we add this to an existing test? I don't think we need an extra test for it but I am also fine with this.

Reply:
I think having a separate test is useful, and we don't exactly have a problem with test runtimes getting excessive here